etc/docs/template/data/releases.toml

```diff
@@ -0,0 +1,107 @@
+current = "4.4 Driver"
+mongoDBManual = "https://docs.mongodb.com/drivers/node/"
+
+[[versions]]
+version = "4.4 Driver"
+status = "current"
+api = "./4.4"
+usesMongoDBManual = true
+semvarVersion = "4.4"
```

Comment:
I noticed that this is `semvar` instead of `semver` is that how it was originally? what were the changes that were made to this file vs the original one?

Reply:
no that's just a typo.  I added semverVersion to each line so that each entry matched the schema in my script but that's the only change in this file